Documentation/ceph-disaster-recovery.md

```diff
@@ -336,3 +336,22 @@ Assuming `dataHostPathData` is `/var/lib/rook`, and the `CephCluster` trying to
 1. Bring the Rook Ceph operator back online by running `kubectl -n rook-ceph edit deploy/rook-ceph-operator` and set `replicas` to `1`.
 1. Watch the operator logs with `kubectl -n rook-ceph logs -f rook-ceph-operator-xxxxxxx`, and wait until the orchestration has settled.
 1. **STATE**: Now the new cluster should be up and running with authentication enabled. `ceph -s` output should not change much comparing to previous steps.
+
+## Adopting into a new Kubernetes cluster with PVCs
```

Comment:
What about this title, and also moving it higher in the document before the section "Adopt an existing Rook Ceph cluster into a new Kubernetes cluster"? I think this will be a more useful/common scenario than the other one. Maybe we need a simple list of links at the top to the three scenarios as well since it's getting longer, similar to the [common issues](https://rook.github.io/docs/rook/v1.4/ceph-common-issues.html) page.
```suggestion
## Backing up and restoring a cluster based on PVCs into a new Kubernetes cluster
```